action.yml

```diff
@@ -38,4 +38,4 @@ branding:
   icon: 'send'
 runs:
   using: 'docker'
-  image: 'Dockerfile'
+  image: 'docker://ghcr.io/wzieba/firebase-distribution-github-action/docker-image:latest'
```

Comment:
I'd vote to use the hard-coded version of the image. In case if we'll introduce breaking change between image and action's entrypoint, we'll break older builds.